packages/projection/src/operators/certificates/withCertificates.ts

```diff
@@ -26,16 +26,18 @@ const blockCertificates = ({
     body
   }
 }: WithBlock) =>
-  body.flatMap(({ body: { certificates = [] } }, txIndex) =>
-    certificates.map((certificate, certIndex) => ({
-      certificate,
-      pointer: {
-        certIndex,
-        slot,
-        txIndex
-      }
-    }))
-  );
+  body
+    .filter((tx) => tx.isValid !== false)
```

Comment:
This is OnChainTx - also need to update core Block type to have OnChainTx[] in the body.
```suggestion
    .filter((tx) => tx.inputSource === 'inputs')
```

Reply:
Done in: 
- [a21d6a4](https://github.com/input-output-hk/cardano-js-sdk/pull/607/commits/a21d6a4e489ae6c979d9f2cdfdc11b0b686192a2)
- [57e3818](https://github.com/input-output-hk/cardano-js-sdk/pull/607/commits/57e381891cbcff43053bac1d1e6611a702085d5f)